kornia/augmentation/augmentation.py

```diff
@@ -23,34 +23,31 @@ def __init__(self, return_transform: bool = False, same_on_batch: bool = False,
 
 
 class RandomHorizontalFlip(AugmentationBase2D):
-    r"""Horizontally flip a tensor image or a batch of tensor images randomly with a given probability.
 
+    r"""Applies a random horizontal flip to a tensor image or a batch of tensor images randomly with a given probability.
```

Comment:
remove last randomly word (I forgot it). Check others too.
`Applies a random horizontal flip to a tensor image or a batch of tensor images with a given probability.`